source/renderer/app/components/wallet/settings/ChangeSpendingPasswordDialog.js

```diff
@@ -149,7 +166,7 @@ export default class ChangeSpendingPasswordDialog extends Component<Props> {
           validators: [
             ({ field, form }) => {
               const spendingPassword = form.$('spendingPassword').value;
-              if (spendingPassword.length === 0) return [true];
+              if (spendingPassword.length === 0) return [false];
```

Comment:
@DeeJayElly I'm not sure that this is correct. If the new password is empty then there is no need to check if the repeated password matches a new password so there can't be an error for that specific case. You will get default error "Error".
And here is proof :
![Screen Shot 2020-04-29 at 14 08 57](https://user-images.githubusercontent.com/18653950/80594273-fdcfda00-8a22-11ea-82ac-2b24a981a9a7.png)